dashboard/api.py

```diff
@@ -29,7 +29,8 @@
 from profiles.api import get_social_auth
 
 # maximum number of exam attempts per payment
-ATTEMPTS_PER_PAID_RUN = 2
+ATTEMPTS_PER_PAID_RUN_OLD = 2
```

Comment:
We might want to leave a comment here to explain what this constant is for. Also, we'll be able to remove it at some point, right?